docs/containers-storage.conf.5.md

```diff
@@ -148,31 +126,70 @@ Sets the log level of devicemapper.
     7: LogLevelDebug
 
 **min_free_space**=""
-
-Specifies the min free space percent in a thin pool required for new device creation to succeed. Valid values are from 0% - 99%. Value 0% disables. (default: 10%)
+  Specifies the min free space percent in a thin pool required for new device creation to succeed. Valid values are from 0% - 99%. Value 0% disables. (default: 10%)
 
 **mkfsarg**=""
+  Specifies extra mkfs arguments to be used when creating the base device.
 
-Specifies extra mkfs arguments to be used when creating the base device.
+**mountopt**=""
+  Comma separated list of default options to be used to mount container images.  Suggested value "nodev".
 
-**use_deferred_deletion**=""
+**size**=""
+  Maximum size of a container image.   This flag can be used to set quota on the size of container images.
 
-Marks thinpool device for deferred deletion. If the thinpool is in use when the driver attempts to delete it, the driver will attempt to delete device every 30 seconds until successful, or when it restarts.  Deferred deletion permanently deletes the device and all data stored in the device will be lost. (default: true).
+**use_deferred_deletion**=""
+  Marks thinpool device for deferred deletion. If the thinpool is in use when the driver attempts to delete it, the driver will attempt to delete device every 30 seconds until successful, or when it restarts.  Deferred deletion permanently deletes the device and all data stored in the device will be lost. (default: true).
 
 **use_deferred_removal**=""
-
-Marks devicemapper block device for deferred removal.  If the device is in use when its driver attempts to remove it, the driver tells the kernel to remove the device as soon as possible.  Note this does not free up the disk space, use deferred deletion to fully remove the thinpool.  (default: true).
+  Marks devicemapper block device for deferred removal.  If the device is in use when its driver attempts to remove it, the driver tells the kernel to remove the device as soon as possible.  Note this does not free up the disk space, use deferred deletion to fully remove the thinpool.  (default: true).
 
 **xfs_nospace_max_retries**=""
+  Specifies the maximum number of retries XFS should attempt to complete IO when ENOSPC (no space) error is returned by underlying storage device. (default: 0, which means to try continuously.)
+
+### STORAGE OPTIONS FOR OVERLAY TABLE
 
-Specifies the maximum number of retries XFS should attempt to complete IO when ENOSPC (no space) error is returned by underlying storage device. (default: 0, which means to try continuously.)
+The `storage.options.overlay` table supports the following options:
+
+**ignore_chown_errors** = "true|False"
```

Comment:
What's the default?  Did you mean to upper case "False" here?

Reply:
Fixed